test/test_ops.py

```diff
@@ -160,6 +160,43 @@ def test_numpy_ref(self, device, dtype, op):
         finally:
             torch.set_default_dtype(cur_default)
 
+    # Tests that the cpu and gpu results are consistent
+    @onlyCUDA
+    @suppress_warnings
+    @ops(_ops_and_refs)
+    def test_compare_cpu(self, device, dtype, op):
+
+        if dtype not in op.dtypes:
```

Comment:
Running this test in every supported dtype seems like a lot. Maybe just once is OK to start with?  The test can ask for "any one" dtype following the instructions here:

https://github.com/pytorch/pytorch/blob/ebd4e90ff7f6d1e57bbf6f2e717a8addb6b28e28/torch/testing/_internal/common_device_type.py#L674

Reply:
done